### PR TITLE
Added tooltip to link

### PR DESCRIPTION
**Pull request by devnoorfatima**
Bugzilla Bug: [XXXXX](https://bugzilla.mozilla.org/show_bug.cgi?id=XXXXX)

### CODE REVIEW

File: ui/src/components/TaskDetailsCard/index.jsx [hunk 1]
```diff
@@ -214,7 +214,9 @@ export default class TaskDetailsCard extends Component {
                   component="a"
                   href={task.metadata.source}
                   target="_blank"
-                  rel="noopener noreferrer">
+                  rel="noopener noreferrer" 
+                  title ="https://hg.mozilla.org/integration/mozilla-inbound/raw-file/8735151820ebeaf7ee06e9f2ed16868c164d85ab/.taskcluster.yml"
```
Comment (djmitche) [inline ui/src/components/TaskDetailsCard/index.jsx]: I think the idea of the issue is that it shows the value of the `source` property for the task -- not that it shows this exact title every time :)

File: package.json [hunk 2]
```diff
@@ -167,6 +167,7 @@
     "zurvan": "^0.5.2"
   },
   "scripts": {
+
```
Comment (djmitche) [inline package.json]: can you remove this stray blank line?

Comment (devnoorfatima) [reply package.json]: I removed this line also run the yarn lint --fix in ui.

File: ui/src/components/TaskDetailsCard/index.jsx (hunk 1 above)
Comment (devnoorfatima) [reply ui/src/components/TaskDetailsCard/index.jsx]: I am sorry what do you mean by value of source property for the task.

Comment (devnoorfatima) [reply ui/src/components/TaskDetailsCard/index.jsx]: Do you mean the title should appear only when Source is hovered.

Comment (djmitche) [reply ui/src/components/TaskDetailsCard/index.jsx]: What you've written will always show the exact same link on hover, for every task.  That's not what we want -- every task will have a different link, and we should show that on hover.

**Comment by devnoorfatima** [comment]
Thanks, You have been really nice and helpful. This is my first time
contributing to open source project so simple things seems difficult to
me.I am not sure which issue to choose next.If you can suggest me some that
I can do.

**Comment by djmitche** [comment]
I think #1487 fits the bill?

**Comment by helfi92** [comment]
@all-contributors please add @devnoorfatima for code.

**Pull request merged**